src/components/ModalPayment/styles.module.scss

```diff
@@ -14,12 +14,22 @@
   }
 }
 
+@media screen and (max-width: 450px) {
+  .headerView {
+    padding-top: 25px;
+    display: flex;
```

Comment:
Why not make it always use flexbox with text-align center?

Reply:
good point :sweat_smile: i'll bring it out